tests/unit/drop-zone.spec.js

```diff
@@ -18,6 +19,8 @@ describe('DropZone', () => {
   beforeEach(async () => {
     w = await localMount(DropZone);
     now = freezeTime();
+    // Mock the clients upload function
+    Client.prototype.upload = jest.fn();
```

Comment:
why `upload` and not `handleUpload?`

Reply:
Because there is no `handleUpload` method on the client.